liquibase-couchbase/src/main/java/liquibase/ext/couchbase/provider/ExpressionDocumentKeyProvider.java

```diff
@@ -1,18 +1,19 @@
 package liquibase.ext.couchbase.provider;
 
+import java.util.Arrays;
```

Comment:
This reformatting result looks like a misalignment between your and project code styles. Pls check your code style or just try to avoid reformatting the whole class in case this produces such a number of line changes without exact relation to the task

Reply:
Yes, it seems so. I tried to use code style from the repo `intellij-java-google-style.xml` but I found there an <option name="INDENT_SIZE" value="2" /> that does not match current project code style at all. Probably most of the guys use default code style instead of `project code style`:)

I changed format of this class